### ISSUE TITLE
Height of element fixed at 400px

### ISSUE THREAD

**Issue by jcrowthe** [Saulis/iron-data-table#5]
At line 111 or iron-data-table.html, I believe the height should be set to 'auto' instead of '400px', as follows:

`:host {
        display: block;
        height: auto;
        position: relative;
        overflow-x: auto;
        overflow-y: hidden;
        -webkit-overflow-scrolling: touch;
      }`

This fits the height of the element to the encapsulating element. I didn't think this merited a pull request though ;)

By the way, I do want to mention how ridiculously excited I am about this element. Keep up the great work, and I will contribute where I can! 

Thanks!

**Comment by Saulis** [comment]
Hi!

First of all, thanks for the kind words :smile: 

The reason why I've left the default height of `400px` is that `iron-list` seems to render quite poorly on the first pass when there isn't any explicit height set. So, it's in a way been a backup value, which can be overridden by the user.

But, I'll have a look if it's just enough to add a comment about that somewhere and use `height: auto`.

`iron-list` has this in their `index.html`:

> > Important: iron-list must either be explicitly sized, or delegate scrolling to an explicitly sized parent. By "explicitly sized", we mean it either has an explicit CSS height property set via a class or inline style, or else is sized by other layout means (e.g. the flex or fit classes).

I'll also try how it would work with `iron-flex-layout` so that I wouldn't need to set any explicit heights.

**Comment by Saulis** [comment]
Couldn't get it working properly without an explicit height on a parent, I'll add a note about this on the docs.

**Comment by jcrowthe** [comment]
I have managed to get the height to properly be set (while using 'height: auto') when I navigate to a page with iron-data-table on it. It is a small nuance, but it functions for what I need.

Thank you for looking into it though! I did too, but was unable to get it to work using iron-flex-layout or the generic CSS flexbox. Unfortunately, it may be something that iron-list needs to find a way to fix before it will work here.
